lib/main_containers.pm

```diff
@@ -45,35 +45,34 @@ sub is_ubuntu_host {
 }
 
 sub load_image_test {
-    my ($runtime) = @_;
-    my $args = OpenQA::Test::RunArgs->new();
-    $args->{runtime} = $runtime;
+    my ($runtime, $run_args) = @_;
+    $run_args->{runtime} = $runtime;
 
-    loadtest('containers/image', run_args => $args, name => "image_$runtime");
+    loadtest('containers/image', run_args => $run_args, name => "image_$runtime");
 }
 
 sub load_3rd_party_image_test {
-    my ($runtime) = @_;
-    my $args = OpenQA::Test::RunArgs->new();
-    $args->{runtime} = $runtime;
+    my ($runtime, $run_args) = @_;
+    $run_args->{runtime} = $runtime;
 
-    loadtest('containers/third_party_images', run_args => $args, name => $runtime . "_3rd_party_images");
+    loadtest('containers/third_party_images', run_args => $run_args, name => $runtime . "_3rd_party_images");
 }
 
 sub load_container_engine_test {
-    my ($runtime) = @_;
-    my $args = OpenQA::Test::RunArgs->new();
-    $args->{runtime} = $runtime;
+    my ($runtime, $run_args) = @_;
+    $run_args->{runtime} = $runtime;
 
-    loadtest('containers/container_engine', run_args => $args, name => $runtime);
+    loadtest('containers/container_engine', run_args => $run_args, name => $runtime);
 }
 
 sub load_image_tests_podman {
-    load_image_test('podman');
+    my ($run_args) = @_;
+    load_image_test('podman', $run_args);
```

Comment:
would it make sense to do this here:
```
$run_args->{runtime} = 'podman';
load_image_test($run_args);
```

and then 
```
sub load_image_test {
    my ($run_args) = @_;
    loadtest('containers/image', run_args => $run_args, name => "image_$runtime");
}
```

WDYT?

Reply:
I did what you suggested as separate commit. Please re-check @jlausuch